pandas/tests/groupby/test_groupby.py

```diff
@@ -2471,7 +2470,6 @@ def test_groupby_series_with_tuple_name():
     tm.assert_series_equal(result, expected)
 
 
-@pytest.mark.xfail(not IS64, reason="GH#38778: fail on 32-bit system")
```

Comment:
I wonder why those are passing. Are we still supporting 32bit?

Reply:
They somehow pass for some reason (and others fail). I'm not sure why (maybe it's the compiler flags?).

It seems as of right now more tests pass than are failing now, but some tests could still be flaky (last 3 times 32bit ran green, though).